scripts/migration/archive_registrations_on_IA.py

```diff
@@ -0,0 +1,28 @@
+import logging
```

Comment:
Why is this in scripts and not management commands?

Reply:
I didn't really consider it, but it wouldn't hurt to make it one.